server/controller/src/main/java/ai/starwhale/mlops/exception/SWValidationException.java

```diff
@@ -13,8 +13,8 @@
 
     static final String PREFIX_TIP="invalid request on subject ";
 
-    private String code;
-    private String tip;
+    private final String code;
+    private final String tip;
```

Comment:
now wo have several similar name：“message”、“tip”、“msg”，use a generic name？

Reply:
"tip" is considered as a user friendly message. "message" and "msg" are more generic than "tip"